codecs.go

```diff
@@ -3,9 +3,8 @@ package rsmt2d
 import "fmt"
 
 const (
-	LeopardFF16 = "LeopardFF16"
-	LeopardFF8  = "LeopardFF8"
-	RSGF8       = "RSFG8"
+	Leopard = "Leopard"
+	RSGF8   = "RSFG8"
```

Comment:
Just noted when reviewing locally that this constant is not used properly. 
This is somewhat orthogonal to this PR but it should be used here IMO: https://github.com/celestiaorg/rsmt2d/blob/866085f9d14cbcf7dad8c3663c5d47d5ffa3621f/infectiousRSGF8.go#L12